client/js/table-model.js

```diff
@@ -16,6 +16,14 @@ class TableModel {
 	setValue(location, value) {
 		this.data[this._getCellId(location)] = value;
 	}
+
+	addRow() {
+		this.numRows++;
```

Comment:
Please use spaces not tabs.  Set your editor accordingly.  Confer with Kay if you aren't sure how to do this.